src/components/CloudHeader/CloudHeader.js

```diff
@@ -0,0 +1,164 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import classNames from 'classnames';
+import CloudHeaderWrapper from './CloudHeaderWrapper';
+import CloudHeaderMenu from './CloudHeaderMenu';
+import CloudHeaderLogo from './CloudHeaderLogo';
+import CloudHeaderList from './CloudHeaderList';
+import CloudHeaderListItem from './CloudHeaderListItem';
+
+export default class CloudHeaderV2 extends React.Component {
+  state = {
+    isMenuOpen: false,
```

Comment:
Should we change `isMenuOpen` -> `isMenuActive` to match the other state value naming conventions?

Reply:
Done